client/components/modal/index.css

```diff
@@ -31,3 +31,8 @@
 .modal-container .message {
   line-height: 30px;
 }
+.load-container {
```

Comment:
I have found out a lot of code which defines similar classes and even a `<Loader>` component, added by different people at different time.
We should put effort in avoiding to define the same thing multiple time. I have done some work in this regard which I will be pushing soon, please follow up with this and let's ensure to define things that are reused across the entire application only once.